docs/gatsby-config.js

```diff
@@ -219,15 +240,30 @@ const sections = [
       "Flow CLI": [
         "docs/index",
         "docs/install",
-        "docs/generate-keys",
+        "docs/initialize-configuration",
+        "docs/security",
+      ],
+      Keys: ["docs/generate-keys"],
+      Accounts: [
         "docs/create-accounts",
-        "docs/send-transactions",
-        "docs/query-transactions",
-        "docs/execute-scripts",
-        "docs/initialize-project",
+        "docs/account-add-contract",
+        "docs/account-update-contract",
+        "docs/account-remove-contract",
+        "docs/account-staking-info",
+      ],
+      "Project Deployment": [
+        "docs/start-emulator",
         "docs/project-contracts",
         "docs/deploy-project-contracts",
       ],
+      Scripts: ["docs/execute-scripts"],
+      Transactions: ["docs/send-transactions", "docs/get-transactions"],
+      "Get Flow data": [
+        "docs/get-blocks",
+        "docs/get-events",
+        "docs/get-accounts",
+        "docs/get-collections",
+      ],
       FAQ: [
```

Comment:
Hmm, is the FAQ intended to be in the CLI section? Can we take it out?

Reply:
FAQ is in all sections.